SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -2098,10 +2098,11 @@ void CFlowOutput::WriteForcesBreakdown(const CConfig* config, const CSolver* flo
           break;
         case TURB_MODEL::SST:
           file << "Menter's SST\n";
-          break;
-        case TURB_MODEL::SST_SUST:
-          file << "Menter's SST with sustaining terms\n";
-          break;
+          // add the submodels here
+          //file << "Menter's SST with sustaining terms\n";
+          if (config->GetSSTParsedOptions().sust)
```

Comment:
This is where I think a string in the options would be useful.
@pcarruscag you mentioned we want to keep the structure light.  Do you see a better way to output these messages without a large if/else structure?

Reply:
hmm ok we need it in more than one place... I'll fix that somehow as I review top to bottom.